parser/ast/ast.go

```diff
@@ -167,3 +167,94 @@ type Visitor interface {
 	// ok returns false to stop visiting.
 	Leave(n Node) (node Node, ok bool)
 }
+
+// GetStmtLabel generates a label for a statement.
+func GetStmtLabel(stmtNode StmtNode) string {
```

Comment:
Why move this function to here?

Reply:
```GetStmtLabel``` should be moved into ast. It is a function of ast. we should reduce the complexity of dependencies.